lib/logflare/backends/adaptor/postgres_adaptor/repo.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Logflare.Backends.Adaptor.PostgresAdaptor.Repo do
```

Comment:
can we add `@moduledoc` and doc tags for each function, so that its easier to understand how the adaptor connects and does migrations.